lib/store/index.js

```diff
@@ -233,16 +233,24 @@ export class Store {
     }
   }
 
-  // Returns the embedded grammar for multilanguage, normal grammar otherwise
-  getEmbeddedGrammar(editor: atom$TextEditor): ?atom$Grammar {
+  /**
+   * Returns the embedded grammar for multilanguage, normal grammar otherwise.
+   *
+   * @param {atom$Point} bufferPosition - If specified, returns the embedded grammar at the buffer
+   *                                      position, the one at the cursor position otherwise.
+   */
+  getEmbeddedGrammar(
+    editor: atom$TextEditor,
+    bufferPosition: ?atom$Point
```

Comment:
if the `bufferPosition` is optional, it is better to put it as `bufferPosition?: atom$Point` which means optional parameter that is an atom point vs `bufferPosition: ?atom$Point` which means parameter is required but can be null, undefined or an atom point

Reply:
Yeah definitely. Thanks for pointing out !